src/js/HeartRateSearch.js

```diff
@@ -18,32 +20,70 @@ const muiThemeWhiteText = getMuiTheme({
 });
 
 
-
 class HeartRateSearch extends Component {
 
   constructor() {
     super();
     this.state = {
       currUser: null,
       userHR: [null],
+      numHR: null,
+      maxHR: null,
       graphVisible: false,
     };
     this.getHR = this.getHR.bind(this);
     this.handleChange = this.handleChange.bind(this);
+    this.formatData = this.formatData.bind(this);
+    this.renderGraph = this.renderGraph.bind(this);
   }
 
   getHR(event) {
     var requestURL = "http://vcm-3576.vm.duke.edu:5000/api/heart_rate/" + this.state.currUser
     axios.get(requestURL).then( (response) => { 
       console.log(response.status);
-      this.setState({userHR: response.data});
+      var gData = this.formatData(response.data)
+      this.setState({
+                      userHR: response.data,
+                      numHR: response.data.length,
+                      maxHR: Math.max(...response.data),
+                      graphData: gData,
```

Comment:
you should just be able to call `this.formatData(response.data)` inline right here!